test/locally-installed-sdk.spec.js

```diff
@@ -245,7 +245,7 @@ describe('Locally Installed Google DataStore Emulator Test', () => {
       })
   });
 
-  it('should start the emulator on localhost when specified only port', () => {
+  it.skip('should start the emulator on localhost when specified only port', () => {
```

Comment:
why do you skip these tests?

Reply:
For some reason they are failing randomly. If you run those tests with `it.only` they all work, but not when they are run together. Any idea what is causing this?

Reply:
I will investigate today